qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss

```diff
@@ -0,0 +1,166 @@
+@use '../helpers/functions' as *;
+
+.qdrant-blog-post {
+  &__header {
+    padding-top: $spacer * 2.5;
+    padding-bottom: $spacer * 1.5;
+    background-color: $neutral-98;
+    text-align: center;
+  }
+
+  &__title {
+    font-size: $spacer * 2;
+    line-height: rem(38);
+    margin-bottom: $spacer;
+    color: $neutral-20;
+  }
+
+  &__about {
+    display: flex;
+    justify-content: center;
+    gap: rem(6);
+    font-size: rem(14);
+    line-height: rem(21);
+    color: $neutral-50;
+    margin-bottom: $spacer * 2.5;
+
+    p {
+      margin-bottom: 0;
+    }
+  }
+
+  &__preview {
+    width: 100%;
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: column;
+    gap: $spacer * 2.5;
+    padding-bottom: $spacer * 3;
+  }
+
+  &__social {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+
+    &-title {
+      font-size: rem(18);
+      line-height: rem(27);
+      font-weight: bold;
+      margin-bottom: 0;
+      padding-bottom: $spacer * 0.5;
+      border-bottom: rem(1) solid $primary-50;
+    }
+
+    &-links {
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+      gap: $spacer * 2;
+    }
+  }
+
+  &__content {
+    p, li {
+      margin-bottom: $spacer * 2;
+    }
+
+    h2 {
+      font-size: $spacer * 1.5;
+      line-height: rem(34);
+      margin-bottom: $spacer;
+    }
+
+    h3 {
+      font-size: rem(20);
+      line-height: rem(30);
+      margin-bottom: $spacer;
+    }
+
+    h4 {
+      font-size: rem(18);
+      line-height: rem(27);
+      margin-bottom: $spacer;
+    }
+
+    img {
+      width: 100%;
+    }
+
+    iframe {
+      margin-bottom: $spacer * 2;
+      width: 100%;
+
+      &[title="YouTube video player"] {
+        height: auto;
+        aspect-ratio: 1.7 / 1;
+      }
+    }
+  }
+
+  @include media-breakpoint-up(xl) {
+    &__header {
+      max-width: rem(920);
+      margin: 0 auto;
+      padding-top: $spacer * 5;
+      padding-bottom: $spacer * 5;
+    }
+
+    &__title {
+      font-size: $spacer * 3.5;
+      line-height: rem(57);
```

Comment:
```suggestion
      line-height: rem(67);
```

The `line-height` in design appears to be set to 67px. Now the header looks a bit too compact.

![Screenshot 2024-04-10 at 12 32 13](https://github.com/qdrant/landing_page/assets/7085263/122b38cc-d594-4aa5-a688-fdf098c3bc7c)

Reply:
done